### PR TITLE
Slight edits and additions

### PR DESCRIPTION
**Pull request by juandlt-csu**
This PR is really just me exploring and adding some smaller things to understand how shiny works. 

- I don't think that I did the gs4_auth right(?). Unless I was supposed to put in my gmail there. 
- I added total staff hours to be displayed in the shiny app and the word doc.
    - I did this by setting total_[x]_text to a global variable since I couldn't access that information outside of the observeEvent. This felt... wrong. Let me know if there is a better/more correct way to do this please!
- Slight edits to .Rmd file.

I think I have a better grasp of how this works and what we are doing. I can be more useful now if we want to add functionality!

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -201,8 +207,12 @@ server <- function(input, output) {
     # write values to reactive rates table
     v$data <- editData(v$data, input$rates_table_cell_edit, 'rates_table')
     
+    # setting total_budget_text to global variable so that we can access it with params
+    total_hours_text <<- paste0("Total Staff Hours: ", comma(sum(v$data[,3])))
```
Comment (ccmothes) [inline app.R:211]: Curious about the reason for the double caret assignment (<<-) here? This is new to me!

File: sow_template.Rmd [hunk 2]
```diff
@@ -45,17 +51,29 @@ rates <- params$rates_data
 **Timeline and Milestones:**
 
 ```{r }
-flextable(tibble(Deadline = "placeholder", Deliverable = "placeholder"))
+flextable(tibble(Deadline = "placeholder", Deliverable = "placeholder")) %>% 
+  autofit() %>%
```
Comment (ccmothes) [inline sow_template.Rmd:56]: Does this fit the table to the page?? I tried this exact thing before but it wasn't working for me.

**Review by ccmothes — APPROVED** [review]
All great edits!

File: app.R (hunk 1 above)
Comment (juandlt-csu) [reply app.R:211]: Double carets let you set a global object within a function. Single carets in a function make local objects within the function only, so I couldn't get this info from this section.

File: sow_template.Rmd (hunk 2 above)
Comment (juandlt-csu) [reply sow_template.Rmd:56]: The table is still in the center of the page and does not go across the width of the page.

**Pull request merged**